compiler/codegen/symbol.cpp

```diff
@@ -1090,6 +1108,53 @@ std::string ArgSymbol::getPythonArgTranslation() {
 
       return res;
     }
+  } else if (t->symbol->hasEitherFlag(FLAG_C_PTR_CLASS, FLAG_REF)) {
```

Comment:
Allowing one to pass a numpy array into an export proc expecting a ref seems strange but I don't think it's inherently wrong